Index: ../4feladat/.idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../4feladat/.idea/.gitignore b/../4feladat/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1614097409653)
+++ b/../4feladat/.idea/.gitignore	(date 1614097409653)
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Datasource local storage ignored files
+/../../../../../../../../../../:\Info\I ev\II felev\Adat Alg\Labor\DSA_lab_Antal_Jozsef\Labor 1\4feladat\.idea/dataSources/
+/dataSources.local.xml
+# Editor-based HTTP Client requests
+/httpRequests/
